lib/prawn/vera_pdf.rb

```diff
@@ -0,0 +1,55 @@
+require 'nokogiri'
+require 'open3'
+
+module Prawn
+  module VeraPdf
```

Comment:
> Or is there some GitHub trick for such things?

Not that I know of. Would love to have a manual option like you described.

> As long as you are not annoyed with getting emails for every 'done' I would continue with this practice.

Not at all. It absolutely makes sense since I left quite a few comments and it's hard to keep track otherwise.